.github/workflows/code_generator.yml

```diff
@@ -63,10 +63,15 @@ jobs:
           git sparse-checkout set runtime/include
           cd ..
           cp -rf ./sdk/runtime/include/* ./frb_rust/src/dart_api_dl/
+          cp -rf ./sdk/runtime/include/* ./frb_dart/dart_api_dl/
```

Comment:
Well I am hesitate whether copying the whole dir *twice* is a good way or not

What about this: Instead of copying to frb_dart, can we throw away it, and *points to* the frb_rust copy whenever we need those files? Given that we only need it at *our* develop time (instead of user's develop time), this should be ok